### PR TITLE
Update configure-team-sync.md

### PR DESCRIPTION
**Pull request by arunkg2787**
Fixes # https://github.com/grafana/support-escalations/issues/6217

Configuring Team Sync : Added automatic removal of users context

**What is this feature?**

[Add a brief description of what the feature or update does.]

**Why do we need this feature?**

[Add a description of the problem the feature is trying to solve.]

**Who is this feature for?**

[Add information on what kind of user the feature is for.]

**Which issue(s) does this PR fix?**:

Fixes #

**Special notes for your reviewer:**

Please check that:
- [ ] It works as expected from a user's perspective.
- [ ] If this is a pre-GA feature, it is behind a feature toggle.
- [ ] The docs are updated, and if this is a [notable improvement](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/#how-to-determine-if-content-belongs-in-a-whats-new-document), it's added to our [What's New](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/) doc.

### CODE REVIEW

**Review by arunkg2787 — COMMENTED** [review]
Modified the alignment

File: docs/sources/setup-grafana/configure-security/configure-team-sync.md [hunk 1]
```diff
@@ -42,21 +42,28 @@ If you have already grouped some users into a team, then you can synchronize tha
...
-1. Click **Add group** to save.
+5. Click **Add group** to save.
 
 > Group matching is case insensitive.
 
+## Automated Removal of Users with LDAP Background Synchronization:
+
+When LDAP is configured with active background synchronization in Grafana, it enables the system to automatically remove users from Grafana teams if they have been removed from the corresponding LDAP groups in the Identity Provider. This mechanism helps ensure that team memberships in Grafana stay in sync with group memberships in the external IdP.
+
+It is essential to understand that this automated removal functionality currently requires the use of LDAP with the active background synchronization feature to be fully functional. For other authentication providers such as OAuth or SAML, this automatic user removal from teams may not be available.
```
Comment (chri2547) [inline docs/sources/setup-grafana/configure-security/configure-team-sync.md:61]:
```suggestion
It is essential to understand that this automated removal functionality currently requires using LDAP with the active background synchronization feature to be fully functional. For other authentication providers, such as OAuth or SAML, this automatic user removal from teams might not be available.
```

File: docs/sources/setup-grafana/configure-security/configure-team-sync.md [hunk 2]
```diff
@@ -42,21 +42,28 @@ If you have already grouped some users into a team, then you can synchronize tha
 {{< figure src="/static/img/docs/enterprise/team_add_external_group.png" class="docs-image--no-shadow docs-image--right" max-width= "600px" >}}
 
 1. In Grafana, navigate to **Administration > Teams**.
-1. Select a team.
-1. Go to the External group sync tab, and click **Add group**.
-1. Insert the value of the group you want to sync with. This becomes the Grafana `GroupID`.
+2. Select a team.
```
Comment (chri2547) [inline docs/sources/setup-grafana/configure-security/configure-team-sync.md:45]:
```suggestion
1. Select a team.
```

File: docs/sources/setup-grafana/configure-security/configure-team-sync.md [hunk 3]
```diff
@@ -42,21 +42,28 @@ If you have already grouped some users into a team, then you can synchronize tha
...
+3. Go to the External group sync tab, and click **Add group**.
```
Comment (chri2547) [inline docs/sources/setup-grafana/configure-security/configure-team-sync.md:46]:
```suggestion
1. Go to the External group sync tab, and click **Add group**.
```

**Review by chri2547 — COMMENTED** [review]
@arunkg2787 Thank you very much for the contribution, I've left some suggestions. Please note that in Markdown, all numbered steps are `1.` Please tag me for a final review and approval. 

In addition, we have versioned docs, so I'm assuming that the added content applies to v9.5 and v10?

One final note, you'll need to run prettier. Please let me know if you need assistance with that.

**Pull request closed without merging**